erpnext_com/www/docs/user/manual/en/stock/articles/stock-received-but-not-billed.md

```diff
@@ -1,9 +1,9 @@
 <!-- add-breadcrumbs -->
```

Comment:
first and 3rd image need cropping from the top. use cmd+shift+5 for setting consistent borders and easy screenshots